# Type: feat
# Subtype: Ex
# English Summary: Added option to generate export
# Chinese Summary: 添加了生成导出的选项
# Entry Number: 22

# === Git Diff Content ===

diff --git a/src/com/google/javascript/refactoring/RefactoringDriver.java b/src/com/google/javascript/refactoring/RefactoringDriver.java
index 1b574d8..b42296a 100644
--- a/src/com/google/javascript/refactoring/RefactoringDriver.java
+++ b/src/com/google/javascript/refactoring/RefactoringDriver.java
@@ -92,6 +92,7 @@ public final class RefactoringDriver {
     options.setCheckSymbols(true);
     options.setCheckTypes(true);
     options.setClosurePass(true);
+    options.setGenerateExports(true);
     options.setPreserveGoogRequires(true);
 
     options.setWarningLevel(DiagnosticGroups.MISSING_REQUIRE, CheckLevel.ERROR);
